README.md

```diff
@@ -1 +1,39 @@
-# Newsweek-Clone
\ No newline at end of file
+# Newsweek Clone
+
+Project Live Version: [https://www.newsweek.com/]
+
+Original Page: [https://www.newsweek.com/]
+
+## Built With ✨
+
```

Comment:
[ ] Please add a project screenshot to the readme docs. It helps improve the professionalism of the project.